source/VMelnalksnis.PaperlessDotNet/Documents/DocumentClient.cs

```diff
@@ -109,6 +110,23 @@ public async IAsyncEnumerable<Document<TFields>> GetAll<TFields>(int pageSize, [
 		return await GetCore<Document<TFields>>(id, cancellationToken).ConfigureAwait(false);
 	}
 
+
+	/// <inheritdoc />
+	public async Task<DocumentContent> Download(int id, bool original, CancellationToken cancellationToken = default)
+	{
+		var response = await _httpClient.GetAsync(original ? Routes.Documents.DownloadUri(id).ToString() :
+															 Routes.Documents.DownloadOriginalUri(id).ToString());
+
+		await response.EnsureSuccessStatusCodeAsync().ConfigureAwait(false);
+
+		var headers = response.Content.Headers;
+
+		return new DocumentContent(
+			content: await response.Content.ReadAsStreamAsync().ConfigureAwait(false),
```

Comment:
I generally don't add the parameter names, either creating local variables with descriptive names, or initializing properties directly depending on the case.
```suggestion
			await response.Content.ReadAsStreamAsync().ConfigureAwait(false),
```